src/metrics.ts

```diff
@@ -0,0 +1,30 @@
+import { FliptApi } from "@flipt-io/flipt";
+import { Evaluation } from "api/resources/evaluation/client/Client";
+
+export class FliptApiClientWithMetrics {
```

Comment:
```suggestion
export class FliptMetrics {
```

how about just simplify the naming?